nsv13/code/modules/munitions/ship_weapons/energy_weapons/phaser.dm

```diff
@@ -81,24 +81,6 @@
 		if(src in weapon_type.weapons["loaded"])
 			LAZYREMOVE(weapon_type.weapons["loaded"] , src)
 
-/obj/machinery/ship_weapon/energy/fire(atom/target, shots = weapon_type.burst_size, manual = TRUE)
```

Comment:
Oh this is what I was talking. Hmmm. Yeah, I don't like either of our solutions, but I guess yours is OK for now.